.github/workflows/deploy_docs.yml

```diff
@@ -9,8 +9,42 @@ on:
 
 jobs:
   build:
-    uses: ./.github/workflows/build_docs.yml
```

Comment:
What's the reason for reason for redefining a build job instead of using this one?

Reply:
I see, I guess one has fast docs for just testing purposes